app/views/admin_public_body_categories/edit.html.erb

```diff
@@ -8,6 +8,7 @@
 
         <div class="form-actions">
           <%= f.submit 'Save', accesskey: 's', class: 'btn btn-success' %>
+          <%= link_to 'Admin page', admin_tag_path(tag: @public_body_category.category_tag), class: 'btn' %>
```

Comment:
This would make it a bit clear that the button relates to the _tag_ admin page:

```diff
diff --git a/app/views/admin_public_body_categories/_form.html.erb b/app/views/admin_public_body_categories/_form.html.erb
index 9c3b9bdcd..37234bde9 100644
--- a/app/views/admin_public_body_categories/_form.html.erb
+++ b/app/views/admin_public_body_categories/_form.html.erb
@@ -62,6 +62,10 @@
          cannot be modified.
        </span>
      <% end %>
+
+     <% if @public_body_category.category_tag %>
+       <%= link_to 'Tag admin page', admin_tag_path(tag: @public_body_category.category_tag), class: 'btn' %>
+     <% end %>
   </div>
 </div>
 
diff --git a/app/views/admin_public_body_categories/edit.html.erb b/app/views/admin_public_body_categories/edit.html.erb
index 4b5b69fdc..2cb268ef3 100644
--- a/app/views/admin_public_body_categories/edit.html.erb
+++ b/app/views/admin_public_body_categories/edit.html.erb
@@ -8,7 +8,6 @@
 
         <div class="form-actions">
           <%= f.submit 'Save', accesskey: 's', class: 'btn btn-success' %>
-          <%= link_to 'Admin page', admin_tag_path(tag: @public_body_category.category_tag), class: 'btn' %>
           <%= link_to 'Public page', list_public_bodies_by_tag_path(@public_body_category.category_tag), class: 'btn' %>
           <%= link_to 'List all', admin_categories_path, class: 'btn' %>
         </div>
```

I did have a quick faff around with [attached buttons](https://getbootstrap.com/2.2.0/base-css.html#forms:~:text=Buttons%20instead%20of%20text) but it added a more complexity because of the existing conditional. Don't think it's high value enough to worry about, but up to you.